sources/platform/workflow/job/runsh.md

```diff
@@ -125,33 +125,34 @@ A description of the job YML structure and the tags available is in the [jobs se
 
     * `TASK` -- Required, at least one script line needs to be present
         * `script:` -- should contain one or more lines of bash script to be
-          executed
+          executed.
         * `runtime:` -- task level runtime options.
```

Comment:
i think this runtime section can also have a `container: true/false`. we should mention that.

Reply:
"defaults to true. If false, the task will be run directly on the host machine."